akka-stream/src/main/scala/akka/stream/stage/GraphStage.scala

```diff
@@ -1893,34 +1826,14 @@ trait OutHandler {
   @throws(classOf[Exception])
   def onPull(): Unit
 
-  /**
-   * Called when the output port will no longer accept any new elements. After this callback no other callbacks will
-   * be called for this port.
-   */
-  @throws(classOf[Exception])
-  @deprecatedOverriding("Override `def onDownstreamFinish(cause: Throwable)`, instead.", since = "2.6.0") // warns when overriding
-  @deprecated("Call onDownstreamFinish with a cancellation cause.", since = "2.6.0") // warns when calling
-  def onDownstreamFinish(): Unit = {
```

Comment:
Wonder if this has been fully migrated everywhere in Akka libraries? We might want to keep this for some longer.

Reply:
That's a problem.

Reply:
Only stream contrib among the repo mains as far as I can see, but still might be best to keep it around for some more time.